modules/ROOT/pages/cassandra/cassandra-connector-xml-maven.adoc

```diff
@@ -0,0 +1,74 @@
+= Cassandra XML and Maven Support - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+If you manually code a Mule application in XML either from the Studio XML editor or standalone, to get access to a connector, you must:
+
+* Add the namespace for the connector to your XML flow.
+
+* Add dependency information to your Maven `pom.xml` file.
+
+In the Studio visual editor, when you drag a connector from the palette to the canvas, Studio automatically populates the XML code with the connector namespace and schema location.
+
+
+== Add a Namespace for the Connector
+If you manually code the Mule application in Studio’s XML editor or another text editor, paste the following into the header of your configuration XML, inside the `<mule>` tag:
+
+[source,xml,linenums]
+----
+http://www.mulesoft.org/schema/mule/cassandra-db
+http://www.mulesoft.org/schema/mule/cassandra-db/current/mule-cassandra-db.xsd
+----
+
+== Configure the Maven POM File
+
+The `pom.xml` is a Maven file that Anypoint Studio generates that contains dependencies for a Mule application. When you code a Mule application manually, include this XML snippet in your `pom.xml` file to enable access to this connector. The Mule runtime translates the `LATEST` dependency keyword to the current version.
```

Comment:
Suggestion:
Anypoint Studio generates a Maven `pom.xml` file, which contains dependencies for a Mule app.
Per Mysti, we can't use "Mule runtime"--it has to be Mule runtime engine, or just Mule.